migrations/versions/bcd74f0a02ea_added_index_for_barcodes_group_id_in_.py

```diff
@@ -0,0 +1,22 @@
+"""Added index for barcodes_group_id in barcodes tables
+
+Revision ID: bcd74f0a02ea
+Revises: e505f2e15499
+Create Date: 2022-10-11 15:23:43.608690
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "bcd74f0a02ea"
+down_revision = "e505f2e15499"
```

Comment:
Is this right?  I don't see an `e505f2e15499` revision in the repository.

Reply:
I think the down_revision - "e505f2e15499" was referring to this migration -migrations/versions/e505f2e15499_add_child_barcode_counter_table.py

Reply:
Wow yes sorry.  I thought my local copy was up to date, but it was quite old.  I see that now.